app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -473,28 +522,46 @@ export default {
         @click="onToggleContactPanel"
       />
     </div>
-    <ul class="conversation-panel">
+    <ul
```

Comment:
Please refrain from modifying any existing logic. Instead, move all elements related to the current change to a new container component. The only change on this component should be a v-if directive for the <ul class=“conversation-panel”> element and on the else case mount the new message bubble container.

Reply:
I can build a ConversationListNext.vue component and add it before the `ul` and toggle them with `v-if`, how does that sound?